Solutions/VMware SD-WAN and SASE/Analytic Rules/vmw-sdwan-device-congestion.yaml

```diff
@@ -0,0 +1,41 @@
+id: a88ead0a-f022-48d6-8f53-e5a164c4c72e
+name: VMware SD-WAN Edge - Device Congestion Alert - Packet Drops
+version: 1.0.0
+kind: Scheduled
+description: The VMware Edge Cloud Orchestrator reported an edge congestion event, where the Edge is dropping a large number of packets on one of its interfaces. This could indicate an ongoing Denial of Service attack against an appliance. Please make sure that Network Flood Protection is turned on.
+severity: Medium
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+requiredDataConnectors:
+  - connectorId: VMwareSDWAN
+    dataTypes:
+      - SDWAN
+tactics:
+- Impact
+relevantTechniques:
+- T1498
+query: |-
+  VMware_VECO_EventLogs_CL
+  | where event == "EDGE_CONGESTED"
+  | where message contains "high number of packet drops"
+  | extend edgeSerialNumber = extract("edgeSerialNumber: (.+)$", 1, detail)
+suppressionEnabled: false
+incidentConfiguration:
+  createIncident: true
+  groupingConfiguration:
+    enabled: true
+    reopenClosedIncident: false
+    lookbackDuration: 5h
+    matchingMethod: AllEntities
+    groupByEntities: []
+    groupByAlertDetails: []
+    groupByCustomDetails: []
+eventGroupingSettings:
+  aggregationKind: SingleAlert
+customDetails:
+  Edge_SerialNr: edgeSerialNumber
+sentinelEntitiesMappings:
```

Comment:
Hello @stlaszlo, Im not getting this sentinelEntitiesMappings
You can entity mappings over here